public/js/chat.js

```diff
@@ -765,16 +765,24 @@ chatInput.addEventListener("keypress", function (event) {
     // Make links clickable
     msg = msg.autoLink();
     // Send message over data channel
+    if (isEmptyOrSpaces(msg) == false) {
     dataChanel.send("mes:" + msg);
-    // Add message to screen
     addMessageToScreen(msg, true);
+
+    }
+    // Add message to screen
```

Comment:
That comment belongs on a different line ;)

Reply:
done :-)